composer.json

```diff
@@ -4,7 +4,7 @@
     },
     "repositories" : [ {
         "type" : "vcs",
-        "url" : "https://github.com/crysalead/kahlan.git"
+        "url" : "https://github.com/jails/kahlan.git"
```

Comment:
The repo is `crysalead/kahlan` but will be transfered to `kahlan/kahlan` for the 3.0. I'll keep you posted ;-)